src/services/ui/src/features/submission/app-k/index.tsx

```diff
@@ -11,18 +11,21 @@ import { useGetUser } from "@/api/useGetUser";
 import { Authority } from "shared-types";
 import { useMutation } from "@tanstack/react-query";
 import { useNavigate } from "@/components/Routing";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 import * as Content from "@/components";
 import { useOriginPath, zAppkWaiverNumberSchema } from "@/utils";
-import { Link } from "react-router-dom";
+import { Link, useLocation } from "react-router-dom";
 import { SlotAdditionalInfo, SlotAttachments } from "@/features";
+import { seaStatusPoller } from "@/utils/Poller/seaStatusPoller";
 
 export const AppKSubmissionForm = () => {
   const nav = useNavigate();
   const crumbs = C.useLocationCrumbs();
   const { data: user } = useGetUser();
   const modal = C.useModalContext();
+  const location = useLocation();
   const originPath = useOriginPath();
+  const [isDataPolling, setIsDataPolling] = useState(false);
```

Comment:
this is required equipment for use of the poller, we should bundle this in a hook exported from that package since that is a widely applicable utility

Reply:
if this state isn't necessary for the new ActionForm pattern (which i believe it isn't), then disregard. my thought was that this would be necessary for all polling 👍🏼